code/modules/reagents/chemistry/recipes/drugs.dm

```diff
@@ -63,6 +63,7 @@
 	if(ismob(holder.my_atom))
 		var/mob/M = holder.my_atom
 		inside_msg = " inside [ADMIN_LOOKUPFLW(M)]"
+		return
```

Comment:
how is this a bug? there's no evidence this is actually a bug

Reply:
The issue I mentioned, as well as the (admittedly outdated) wiki. This tells me that at some point, meth did not explode in the body. I cannot find a PR or anything that intended to explicitly change this being a thing, which leads be to believe that it's a bug or oversight, and not intended.

Reply:
see prior statement, the issue at no point clarifies how this is a bug

by default we intend all reactions to occur in the body unless they've been explicitly tagged as no mob reaction

Reply:
Just found a PR that was supposed to [remove meth explosion syringes](https://github.com/tgstation/tgstation/pull/29912), which was approved and merged. This was back in 2017, and while I did not play back then, I do know that chem worked quite differently, so presumably whatever change they made to remove them does not apply anymore. From what I can find, there has been no PR since that intended to re-add these.

Reply:
> by default we intend all reactions to occur in the body unless they've been explicitly tagged as no mob reaction

https://github.com/tgstation/tgstation/blob/a7e76abd1a6294d9d57328803dfefdc4f286facb/code/modules/reagents/chemistry/recipes/pyrotechnics.dm#L224C2-L224C2

Reply:
https://github.com/tgstation/tgstation/blob/a7e76abd1a6294d9d57328803dfefdc4f286facb/code/modules/reagents/chemistry/recipes/pyrotechnics.dm#L224

Not saying it's a bug or not but this tag in the pyrotechnics file suggests previous intentions.

Reply:
As per Vladin this is a bug